### DISCUSSION TITLE
cambio sistema di accesso da esterno verso NGINX

### DISCUSSION THREAD

**Discussion by docoscar** [indomus/forum#3163] [Networking]
Salve, 
per vari motivi decido di passare dall'utilizzo di altro verso NGINX che sui vari forum ho visto essere il più sicuro.
Ho peraltro cambiato router.
La situazione attuale è la seguente:
- ho HA esposto e correttamente raggiungibile tramite indirizzo esterno fornito da freedns, per cui se punto a miodominio.freedns:8123 raggiungo il mio HA,
- con il vecchio router e i vecchi servizi che si appoggiavano a let's encrypt di cui ho ancora un certificato valido, riesco a integrare Alexa
- ho un forwarding che funziona perfettamente direttamente su 8123
- se provo ad avviare NGINX mi dice che la porta 443 è utilizzata anche se non so chi la utilizzi.
Vi sarei grato se mi aiutaste a riconfigurare tutto.
Grazie anticipatamente.

**Comment by indomus** [comment]
Noi ci siamo concentrati su Caddy, piuttosto che NGINX. Sw vuoi puoi prendere spunto [da questa guida](https://indomus.it/guide/installare-e-configurare-caddy-su-home-assistant-os-hassos-hassio/), i concetti sono i medesimi.
